collection_size.go

```diff
@@ -22,6 +22,97 @@ type stats struct {
 	Size  int32 `bson:"size"`
 }
 
+// NewDBCollectionsSizeCollector returns a new collector that monitors sizes and object counts
+// of all collections in the specified database.
+func NewDBCollectionsSizeCollector(namespace string, db *mgo.Database) *dbCollectionsSizeCollector {
```

Comment:
i think passing in the db is ok.. unless you have a very strong reason not to do so..